variants/frontend-react-typescript/template.rb

```diff
@@ -15,14 +15,20 @@
   react-dom
 ].map { |name| "@types/#{name}" }
 
+installed_jest_major_version = `yarn list --pattern jest`
```

Comment:
You should be able to statically check this by grabbing the `version` property from `node_modules/jest/package.json`, which imo feels a bit nicer

Reply:
(fwiw this is how we do it in `eslint-plugin-jest`)

Reply:
Nice. will change.